Fwk/AppFwk/cafPdmScripting/cafPdmFieldIOScriptability.h

```diff
@@ -330,7 +330,7 @@ struct PdmFieldScriptabilityIOHandler<DataType*>
         fieldValue = nullptr; // Default initialized to nullptr
 
         QString fieldString;
-        PdmFieldScriptabilityIOHandler<QString>::writeToField( fieldString, inputStream, errorMessageContainer, true );
+        PdmFieldScriptabilityIOHandler<QString>::writeToField( fieldString, inputStream, errorMessageContainer, false );
```

Comment:
Consider creating a local variable like `bool stringsAreQuoted = false`, and use this instead of a direct bool. This makes the code more readable.

Reply:
Agree. Fixed.